### PR TITLE
Dockerized

### PR DESCRIPTION
**Pull request by Deseteral**

### CODE REVIEW

File: fly.toml [hunk 1]
```diff
@@ -11,15 +6,9 @@ kill_timeout = "5s"
 [experimental]
   auto_rollback = true
 
-[build]
```
Comment (gorzalczany) [inline fly.toml:14]: So now we are leveraging default behavior of [looking for dockerfile in application root](https://fly.io/docs/reference/configuration/#specify-a-dockerfile)? Don't we want to describe it explicitely not to leave space for guesses?

Comment (Deseteral) [reply fly.toml:14]: Sure, we could explicitly specify it. I don't have a preference here.

Comment (gorzalczany) [reply fly.toml:14]: Also fly's gotchas:
> When specifying a local Dockerfile, make sure it’s not excluded from the Docker build context in your .dockerignore.

Comment (Deseteral) [reply fly.toml:14]: Done

**Review by gorzalczany — APPROVED** [review]

**Review by gorzalczany — APPROVED** [review]

**Pull request merged**